src/utils/simulateMessageHelper.ts

```diff
@@ -0,0 +1,136 @@
+import * as vscode from 'vscode';
+import { Logger } from '../logger';
+import { loggableAskError } from '../exceptions';
+import { getAvailableLocales } from '../utils/skillHelper';
+import {
+    SKILL_ACTION_URLS,
+    ERRORS,SIMULATOR_WEBVIEW_MESSAGES, SIMULATOR_MESSAGE_TYPE, EN_US_LOCALE
+} from '../constants';
+import * as simulateSkillHelper from '../utils/simulateSkillHelper';
+import * as simulateNonSkillHelper from '../utils/simulateNonSkillHelper';
+
+export let currentLocale = EN_US_LOCALE;
+export let currentSkillId: string;
+export let isSkillEnabled = false;
+
+/**
+ * Handle message sent from Webview when skill status is changed or check skill status 
+ * @param webviewMessage message sent from Webview
+ * @param profile user profile
+ * @param skillId Alexa Skill ID
+ * @param context VSCode extension context
+ */
+export async function handleSkillStatusMessageFromWebview(webviewMessage: Record<string, any>, profile: string, skillId: string,
+    context: vscode.ExtensionContext): Promise<string> {
+    Logger.verbose(`Calling method: simulateSkillHelper.handleSkillStatusMessageFromWebview`);
```

Comment:
nit: You will need to change these logger info messages too since these methods are now in a different file.